src/styles/sidebar/components/FilterSelect.scss

```diff
@@ -2,6 +2,13 @@
 @use "../../variables" as var;
 
 .FilterSelect {
+  &__menu {
```

Comment:
Might I suggest considering relative height units here (e.g. `vh` units) so that this rule will scale to different viewport sizes? i.e. what happens on phones and tablets? We'll need to handle various viewport sizes that may be shorter than `640px`.

I'd also suggest perhaps purposely breaking in the _middle_ of a `MenuItem` as that makes the discoverability that things scroll easier in interfaces that don't render scroll bars.